option_test.go

```diff
@@ -1,6 +1,7 @@
 package patron
 
 import (
+	http2 "net/http"
```

Comment:
when there is a package name conflict "http" the std package one stays the same and the custom package has to be aliased